extension/chrome/elements/pgp_block.htm

```diff
@@ -13,8 +13,10 @@
 
 <body style="background-color: transparent;">
   <div id="pgp_background" class="pgp_secure">
-    <div id="pgp_signature" data-test="pgp-signature">
-      <div class="cursive"><span></span></div>
+    <div id="pgp_signature" data-test="pgp-signature" class="pgp_badge short">
+      <div class="result"></div>
```

Comment:
also not sure whether we need `result` divs

Reply:
we probably don't